src/run.ts

```diff
@@ -53,6 +54,11 @@ const validateAddresses = async (config) => {
     for (const network in config.WALLETS) {
         const { ADDRESS, SECRET } = config.WALLETS[network];
 
+        if (network != 'ETH' && config.WALLETS['ETH'].ADDRESS && compareAddress(config.WALLETS['ETH'].ADDRESS, ADDRESS)) {
```

Comment:
if `config.WALLETS['ETH']` is undefined the expression will throw an unhandled exception.